src/nunavut/lang/c/__init__.py

```diff
@@ -810,3 +810,16 @@ def filter_full_reference_name(language: Language, t: pydsdl.CompositeType) -> s
         return filter_id(language, not_stropped)
     else:
         return not_stropped
+
+@template_language_filter(__name__)
+def filter_to_standard_bit_length(language: Language, t: pydsdl.PrimitiveType):
```

Comment:
Idk if there's another, existing way to do this. I couldn't find on in lang/c anyways.